.github/workflows/ca-clone-hsm-test.yml

```diff
@@ -114,6 +116,8 @@ jobs:
 
           docker cp id_ed25519.pub primary:/root/.ssh
 
+          docker exec primary chmod 644   /root/.ssh/id_ed25519.pub
+
```

Comment:
Is this needed?

Reply:
Same thing. I will retry without those two things.